src/Pages/ViewList.js

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import { useCollection } from 'react-firebase-hooks/firestore';
 import { db } from '../lib/firebase';
 
-const ViewList = () => {
-  const [value, loading, error] = useCollection(db.collection('lists'));
+const ViewList = ({ token }) => {
+  const [value, loading, error] = useCollection(db.collection(token));
```

Comment:
One thing that could be nice to see here is a better variable name than "value". Throughout the rest of this component, it isn't really clear what "value" is unless you scroll up and see that it's something coming from useCollection here. Perhaps we could rename it to something like "list"

Reply:
@tannaurus Agree! I'll make that change right now.